### PR TITLE
Start FAQ

### PR DESCRIPTION
**Pull request by vweevers**
Mainly to have a central place to explain `abstract-level` and how to upgrade to that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +38,48 @@ At the core of Level are simple key-value stores that follow the characteristics
...
 
-The [`level`](https://github.com/Level/level) module is the recommended way to get started. Visit [`awesome`](https://github.com/Level/awesome) to discover more modules. See our [project board](https://github.com/orgs/Level/projects/2) to find out what we're working on. If you need help - technical, philosophical or other - feel free to open an issue in [`community`](https://github.com/Level/community) or a more specific repository.
+## FAQ
+
+### Where do I start?
+
+The [`level`](https://github.com/Level/level) module is the recommended way to get started. It offers a persistent database that works in Node.js and browsers. To store data in a different way, you might like:
+
+- `level-mem` to keep data in-memory (Node.js and browsers)
+- `level-rocksdb` to use RocksDB instead of LevelDB (Node.js only).
```
Comment (juliangruber) [inline README.md:48]: Do we want to feature rocksdb like this? Is it the 2nd stable integration?

File: README.md [hunk 2]
```diff
@@ -14,7 +38,48 @@ At the core of Level are simple key-value stores that follow the characteristics
...
+
+Visit [`awesome`](https://github.com/Level/awesome) to discover more modules.
```
Comment (juliangruber) [inline README.md:50]:
```suggestion
Visit [`level/awesome`](https://github.com/Level/awesome) to discover more modules.
```

File: README.md [hunk 3]
```diff
@@ -14,7 +38,48 @@ At the core of Level are simple key-value stores that follow the characteristics
...
+
+### What is `abstract-level`?
+
+_If you are new to Level, there is a quick answer: `abstract-level` is the new core of Level on top of which several databases are (or will be) implemented. If you're already familiar with Level modules (before 2022) and have used `level`, `levelup`, `abstract-leveldown`, `encoding-down` or `deferred-leveldown`, read on._
```
Comment (juliangruber) [inline README.md:54]: And if I haven't used these modules before, shall I skip?

File: README.md [hunk 4]
```diff
@@ -14,7 +38,48 @@ At the core of Level are simple key-value stores that follow the characteristics
...
+
+Back in 2012, [`levelup`](https://github.com/Level/levelup) offered a Node.js binding for Google's LevelDB. Authored by Rod Vagg, `levelup` exposed the features of LevelDB in a Node.js-friendly way. It had streams, binary support, encodings... all the goodies. Later on, the binding was moved to `leveldown`, so that other stores could be swapped in while retaining the friendly API of `levelup`.
+
+This is when "up" vs "down" naming was born, where databases followed the formula of "level = levelup + leveldown". For example, `level-mem` was a comvenience package that bundled `levelup` with `memdown`. The `abstract-leveldown` module offered a lower-level abstraction for the "down" part, to encapsulate common logic between "down" stores. Many such stores were written, replacing LevelDB with IndexedDB, RocksDB, in-memory red-black trees, relational databases and more.
```
Comment (juliangruber) [inline README.md:58]:
```suggestion
This is when "up" vs "down" naming was born, where databases followed the formula of "level = levelup + leveldown". For example, `level-mem` was a convenience package that bundled `levelup` with `memdown`. The `abstract-leveldown` module offered a lower-level abstraction for the "down" part, to encapsulate common logic between "down" stores. Many such stores were written, replacing LevelDB with IndexedDB, RocksDB, in-memory red-black trees, relational databases and more.
```

**Review by juliangruber — CHANGES_REQUESTED** [review]
nice going! 👏

File: README.md (hunk 1 above)
Comment (vweevers) [reply README.md:48]: Yeah, maybe not. It's stable, but less popular and often lagging in maintenance.

I'll rephrase the second paragraph to explain that there are alternatives to `level`, using only `level-mem` as an example

File: README.md (hunk 3 above)
Comment (vweevers) [reply README.md:54]: Good point, that should be stated explicitly

**Review by juliangruber — APPROVED** [review]
Great stuff! Would be great to get another review too

**Pull request merged**